Amplify/DevMenu/View/DevMenuList.swift

```diff
@@ -0,0 +1,54 @@
+//
+// Copyright 2018-2020 Amazon.com,
+// Inc. or its affiliates. All Rights Reserved.
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+
+import SwiftUI
+
+/// View containing a list of developer menu items
+@available(iOS 13.0.0, *)
+struct DevMenuList: View {
+
+    /// Integer tags to identify row items in Dev Menu
+    public static let tagEnvironmentInformation = 0
+    public static let tagDeviceInformation = 1
+    public static let tagViewLogs = 2
+    public static let tagFileIssue = 3
```

Comment:
maybe this could become an `enum`?

```swift
enum DeviceInfoLabel {
    case deviceInfo
    case environmentInfo
    // ...

    var stringValue: String {
        switch self {
            case .deviceInfo:
                return "Environment information"
        }
    }
}
```

You can follow the same pattern for the environment/device info labels/values.